bin/encore.js

```diff
@@ -56,6 +57,13 @@ if (runtimeConfig.helpRequested) {
 }
 
 if (runtimeConfig.useDevServer) {
+    try {
+        featuresHelper.ensurePackagesExistAndAreCorrectVersion('webpack-dev-server', 'the webpack Development Server');
+    } catch (e) {
+        console.log(e);
+        process.exit(1);
+    }
```

Comment:
I could have used `isPackageInstalled()`, but I would have to rewrite a lot of logic here to enforce the version, generate the good command, etc... 

Adding `method` parameter to `featuresHelper.ensurePackagesExistAndAreCorrectVersion` was a lot easier and _clean_.